app/src/main/res/layout/otp_dialog.xml

```diff
@@ -17,7 +17,7 @@
         android:inputType="number"
         android:textColor="@color/line_colors"
         app:itemCount="6"
-        app:itemRadius="5dp"
+        app:itemRadius="20dp"
```

Comment:
Doesn't this look better with a lesser corner radius?

Reply:
> I didn't see a Repository for the Vendor, where the fetching of inventory etc would be done? Has that intentionally not been done at this point?

I have left it out on purpose becuase I wasnt sure if the add to inventory way was final. I'll add it if its final.